### PR TITLE
CASMINST-4219 - Add instructions to merge in new CSM 1.2 VCS content to upgrade procedure

### PR DESCRIPTION
**Pull request by spillerc-hpe**
## Summary and Scope

A CSM upgrade can introduce new content to the CSM configuration management repository.

It is necessary to merge this content into any working branch then update the CFS configuration used for NCN personalization to ensure this new Ansible content is applied.

## Issues and Related PRs

* Resolves [CASMINST-4219](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4219)

### Tested on:

  * `hela`

### Test description:

Procedure tested on hela.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [X] Target branch correct
- [ ] CHANGELOG.md updated
- [X] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: upgrade/1.2/Stage_3.md [hunk 1]
```diff
@@ -9,5 +11,80 @@ Run `csm-upgrade.sh` to deploy upgraded CSM applications and services:
...
+   ncn-m002# VCS_PASSWORD=$(kubectl -n services get secret vcs-user-credentials -o jsonpath='{.data.vcs_password}' | base64 --decode)
+   ```
+
+1. Clone the CSM configuration management repository.
+
+   ```bash
+   ncn-m002# git clone https://${VCS_USER}:${VCS_PASSWORD}@api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+   ```
+
+1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:29]:
```suggestion
1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap
```
```suggestion
1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap.
```

File: upgrade/1.2/Stage_3.md [hunk 2]
```diff
@@ -9,5 +11,80 @@ Run `csm-upgrade.sh` to deploy upgraded CSM applications and services:
 ```bash
 ncn-m002# /usr/share/doc/csm/upgrade/1.2/scripts/upgrade/csm-upgrade.sh
 ```
+## Update the CSM configuration management repository
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:14]:
```suggestion
## Update the CSM Configuration Management Repository
```

Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:14]:
```suggestion
## Update the CSM Configuration Management Repository
```

File: upgrade/1.2/Stage_3.md [hunk 3]
```diff
@@ -9,5 +11,80 @@ Run `csm-upgrade.sh` to deploy upgraded CSM applications and services:
...
+
+1. Clone the CSM configuration management repository.
+
+   ```bash
+   ncn-m002# git clone https://${VCS_USER}:${VCS_PASSWORD}@api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+   ```
+
+1. Obtain import_branch of new release from the cray-product-catalog Kubernetes ConfigMap
+
+   In the below example the import branch is `cray/csm/1.9.21`
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:31]:
```suggestion
   In the following example, the import branch is `cray/csm/1.9.21`.
```

Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:31]:
```suggestion
   In the following example, the import branch is `cray/csm/1.9.21`.
```

File: upgrade/1.2/Stage_3.md [hunk 4]
```diff
@@ -9,5 +11,80 @@ Run `csm-upgrade.sh` to deploy upgraded CSM applications and services:
...
+1. Capture most recent commit to use to update the CFS configuration and perform NCN Personalization.
+
+   ```bash
+   ncn-m002# git rev-parse --verify HEAD
+   5616de1ffd375ac94a0f9c51f65dee3063b6be6e
+   ```
+
+1. Update the CFS configuration and perform NCN personalization.
+
+   Please follow the procedure from [Perform\_NCN\_Personalization.md](../../operations/CSM_product_management/Perform_NCN_Personalization.md) to update the CFS configuration and run NCN Personalization.
```
Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:86]:
```suggestion
   Refer to the [Perform NCN Personalization](../../operations/CSM_product_management/Perform_NCN_Personalization.md) procedure to update the CFS configuration and run NCN Personalization.
```

Comment (nrockershousen) [inline upgrade/1.2/Stage_3.md:86]:
```suggestion
   Refer to the [Perform NCN Personalization](../../operations/CSM_product_management/Perform_NCN_Personalization.md) procedure to update the CFS configuration and run NCN Personalization.
```

**Review by gbaker-hpe — APPROVED** [review]

**Review by rkleinman-hpe — CHANGES_REQUESTED** [review]
Please hold off on merging this until we determine if CASMCMS-7890 will make it unnecessary.

**Comment by spillerc-hpe** [comment]
Closing as this will be handled via [CASMINST-4218](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4218)

**Pull request closed without merging**